main/test/flix/Test.Exp.Regions.flix

```diff
@@ -1384,11 +1384,20 @@ namespace Test/Exp/Regions {
         }
 
     @test
-     def testRegionExit01(): Bool \ IO = region r1 {
-         let x  = ref false @ r1;
-         region r2 {
-             spawn { Thread.sleep(Time/Duration.fromSeconds(1)); x := true } @ r2
-         };
-         deref x
-     }
+    def testRegionExit01(): Bool \ IO = region r1 {
+        let x  = ref false @ r1;
+        region r2 {
+            spawn { Thread.sleep(Time/Duration.fromSeconds(1)); x := true } @ r2
```

Comment:
The sleep 1second test is not OK with me. Either reduce it to a much shorter time or drop the test.

Reply:
You could maybe use a channel as sync. point.

Reply:
For reference this test isn't actually anything to do with this PR: it's only showing up as a change because I took the opportunity to fix some broken formatting.

As for how to fix this, the whole point of the test is to ensure that scope exit calls `join`. I don't think I can add any additional synchronisation without missing the point of the test. So I'll reduce the delay. There needs to be *some* delay though or the test could pass just because the thread executes "quickly".